.travis.yml

```diff
@@ -14,17 +14,26 @@ install:
 script:
   - tox
 deploy:
-  provider: pypi
-  user: theforeman
-  password:
-    secure: HmlFYR1Nn325ZYPTWOorcw2qSQNrpxOiEmi3sd3EhVNlk0sR0582ujNY2cLquLxQkFFs3cjDdUVz1FNHC0PX5nOSbA7P0Ks4xZoU2MeXq+HLlreB2w2qcp2XfJXbSFh8ZfqoMCw83lsHkTBwkCYpy/jF6FNUSiP0hWp0aYNMVKwaLB2V8YggkyYStUqtLyq8BlKP1bxWqZTjVia9sk5ZpzG+sDMC/rzwlYDRzk6u9UFITmav6ONCFIsiwBuOTa9KcON21PQGJOBnP3HJB3iyPcGKI/3Ln5aUKVn36tN5hNu3s8VkqVumpxZir5qpqplsJdpY9tRwUTGyVP+CVA8wjxJYP3ziRQpJeHXVdDQ3pU1t9uOPp5Zgfp1Xbbl5QhcAQkHedQ27Xc3WlKdgXZO4dQpTnuFPW71j/J4EM29M1/0RecSYrVvefqWk7ZhIgX4VHydzM9n7VJcIGjXkY5JIWTgEwbXsw9E58iktefYy7+q/z059e0m3+vLJmuWIE+eucmN9pWdunS6GEL8fq9XDL5egtf0c//+mrjHj4u90+8w+TjzdVduqQyQVI3MZ68eM+Wlz0ChjOk32l2N9CmNjGXRFog2YbazZaJT0Z+mRMUlyQUNr6nzC4smsqX8UqzrdHI/g6s9klNoNWdjTF4afewElkQtX9NT0m5SVWAr12hg=
-  on:
-    tags: true
-    python: '3.6'
+  - provider: pypi
+    user: theforeman
+    password:
+      secure: HmlFYR1Nn325ZYPTWOorcw2qSQNrpxOiEmi3sd3EhVNlk0sR0582ujNY2cLquLxQkFFs3cjDdUVz1FNHC0PX5nOSbA7P0Ks4xZoU2MeXq+HLlreB2w2qcp2XfJXbSFh8ZfqoMCw83lsHkTBwkCYpy/jF6FNUSiP0hWp0aYNMVKwaLB2V8YggkyYStUqtLyq8BlKP1bxWqZTjVia9sk5ZpzG+sDMC/rzwlYDRzk6u9UFITmav6ONCFIsiwBuOTa9KcON21PQGJOBnP3HJB3iyPcGKI/3Ln5aUKVn36tN5hNu3s8VkqVumpxZir5qpqplsJdpY9tRwUTGyVP+CVA8wjxJYP3ziRQpJeHXVdDQ3pU1t9uOPp5Zgfp1Xbbl5QhcAQkHedQ27Xc3WlKdgXZO4dQpTnuFPW71j/J4EM29M1/0RecSYrVvefqWk7ZhIgX4VHydzM9n7VJcIGjXkY5JIWTgEwbXsw9E58iktefYy7+q/z059e0m3+vLJmuWIE+eucmN9pWdunS6GEL8fq9XDL5egtf0c//+mrjHj4u90+8w+TjzdVduqQyQVI3MZ68eM+Wlz0ChjOk32l2N9CmNjGXRFog2YbazZaJT0Z+mRMUlyQUNr6nzC4smsqX8UqzrdHI/g6s9klNoNWdjTF4afewElkQtX9NT0m5SVWAr12hg=
+    on:
+      tags: true
+      python: '3.6'
+  - provider: script
+    script: bash .travis/deploy-container.sh 0.2.0
+    on:
+      python: '3.6'
+      branch: 'add-dockerfile'
 script:
   - tox
 addons:
   apt:
     packages:
     - git-annex
     - rpm
+env:
+  global:
```

Comment:
Is this global part needed? Can't you set the env on the actual deploy?
```yaml
  - provider: script
    script: bash .travis/deploy-container.sh 0.2.0
    on:
      python: '3.6'
      branch: 'add-dockerfile'
    env:
      - secure: ...
      - secure: ...
```

Reply:
Have you seen an example of that? I couldn't find the docs.